src/lib/converter/symbols.ts

```diff
@@ -711,7 +712,9 @@ function convertAlias(
             context.resolveAliasedSymbol(symbol),
             exportSymbol ?? symbol
         );
-    } else {
+    } else if (
```

Comment:
This isn't the right check here - it should be `if (!reflection || disableAliases)` since otherwise the reflection will be missing.

Reply:
True, I forgot about that

Reply:
Sorry, that's not what I meant.... that should have been added to the if block on line 708, not down here. This should still just be an else.

Reply:
Ok, now I've clearer picture of this function's purpose.

Although in which cases reflection might be null while converting alias?

Reply:
The reflection will be undefined if the symbol that the alias is pointing to hasn't been converted yet

Reply:
Thanks for pointing me in the right direction :smile: